packages/editor/src/components/post-taxonomies/most-used-terms.js

```diff
@@ -62,8 +62,7 @@ export default function MostUsedTerms( { onSelect, taxonomy } ) {
 				{ terms.map( ( term ) => (
 					<li key={ term.id }>
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>
<img width="271" alt="Most used term before" src="https://github.com/user-attachments/assets/583925dd-63de-461b-9659-83601af67d3c">

</td>
    <td>
<img width="271" alt="Most used term after" src="https://github.com/user-attachments/assets/7aaa43cb-b206-400e-a015-e13a9ec37166">


</td>
  </tr>
</table>
Note: Button variant set to link.